user/pages/03.commerce2/02.developer-guide/02.install-update/01.requirements/docs.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Requirements
+taxonomy:
+    category: docs
+---
+
+ ### Requirements
+
+ -Commerce 2.x requires Drupal 8.5.0 or newer.
```

Comment:
Maybe we should add:

"Generally we require each minor release, as it contains improvements that we use, or to reduce our code base."